libs/core/algorithms/include/hpx/parallel/algorithms/uninitialized_fill.hpp

```diff
@@ -252,11 +245,12 @@ namespace hpx::parallel {
                 partition_result_type>::
                 call(
                     HPX_FORWARD(ExPolicy, policy), first, count,
-                    [value, tok](Iter it, std::size_t part_size) mutable
+                    [value, tok, policy](Iter it, std::size_t part_size) mutable
```

Comment:
Is `tok` still used?

Reply:
vestigial tok had been used in a couple of places as I couldn't find the appropriate loop without tok, let me look into this one more time. Do you suggest implementing such a loop (without using tok) if it doesn't exist?

Reply:
Well, you captured `tok` without using it in the lambda, that's why I asked.

Reply:
my bad, didn't realise that.